blueprints/cloud-operations/uptime-monitoring-tool/source/net-mon-agent/monitoring/utility.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.api.metric_pb2 import MetricDescriptor
+from google.cloud import monitoring_v3
```

Comment:
this is personal preference, but I find that using `requests` and calling API methods directly yields much more readable and efficient code...

Reply:
Updated code using request (as per compute quota script)